File: Benchmarking_Languages_Project/Scala_Tests/Test11.scala
var timing = DenseVector.zeros[Long](15)
for(i <- 0 until 15){
	
	val start = System.currentTimeMillis()
	val T11 = DenseVector.randi(1000,3500000)
	val phi = 1.618033988749895
	val T11_1 = (phi:^T11 - (-phi):^(-T11))/sqrt(5.0)	
	val finish = System.currentTimeMillis()
	timing(i) = (finish - start)
}

println(mean(timing)/1000)

File: Benchmarking_Languages_Project/Julia_Tests/Test11.jl
tic()
for j in 1:15
T11 = randi((100,1000),3500000)
phi = 1.618033988749895
fib = Array(Float64,3500000)
for i in 1:3500000
fib[i] = (phi^T11[i] - (-phi)^(-T11[i]))/sqrt(5)
end
end
timing = toc()
println (timing/15)
